examples/kelvin_helmholtz_instability.jl

```diff
@@ -4,7 +4,7 @@
 
 using Oceananigans
 
-grid = RegularCartesianGrid(size=(64, 1, 64), x=(-5, 5), y=(0, 1), z=(-3, 3),
+grid = RegularCartesianGrid(size=(64, 1, 65), x=(-5, 5), y=(0, 1), z=(-5, 5),
```

Comment:
Why 65? I just want to note that face-centered and cell-centered fields have different lengths when a direction is `Bounded`. So `Nz=64` with bounded vertical directions implies that, in fact, `w` has 65 cells... or maybe this is just to make the grid spacing nice?

Reply:
I only changed that so that the Ri(z) has a point at z=0 demonstrating that it takes the prescribed `Ri` value at that point!

But probably this was bad thinking indeed... Both U(z) and B(z) are plotted using the `z = znodes(Cell, grid)` which is not quite right...

Two questions then:
- we want to have grid points such that indeed the Ri(z) felt by the dynamics includes the `z=0` point; is `Nz=64` ok for this?
- how should we properly plot the base state? should we use the grids that U(z) and B(z) use? I guess so... Then what about Ri(z)?

Reply:
`z = znodes(Cell, grid)` is correct for `u` and `b`, but you're right it's wrong for Ri. We need `zRi = znodes(Face, grid)`.

I'm not sure about dynamics. I don't think there should be a huge difference between odd / even grid points; the discrete approximation converges regardless?

Reply:
ok, I'll fix it tomorrow or have a go ;)